pennylane/ops/qubit/observables.py

```diff
@@ -161,6 +162,52 @@ def eigvals(self):
         """
         return self.eigendecomposition["eigval"]
 
+    @staticmethod
+    def compute_decomposition(A, wires):  # pylint: disable=arguments-differ
+        r"""Decomposes an observable as a sum of Pauli operators (static method).
+
+        Args:
+            A (array or Sequence): hermitian matrix
+            wires (Iterable[Any], Wires): wires that the operator acts on
+        Returns:
+            list[.PauliSentence]: decomposition of the hermitian matrix
+
+        **Examples**
+
+        >>> op = qml.X(0) + qml.Y(1) + 2 * qml.X(0) @ qml.Z(3)
+        >>> op_matrix = qml.matrix(op)
+        >>> qml.Hermitian.compute_decomposition(op_matrix, wires=['a', 'b', 'aux'])
+        [1.0 * Y(b)
```

Comment:
Is this repr correct? I think it should be `Y("b")` - we've tried to make it copy-pasteable (assuming you do `from pennylane import X, Y, Z, I`).

Reply:
If how a pauli sentence is rendered, but the operator is `Y('b')`. So once we adjust the return to be a list with an operator in it, the repr will be `
Y('b') + X('a') + (2 * X('a')) @ Z('aux')`.